docker-compose.yml

```
@@ -120,6 +120,7 @@ services:
       # If you have high volume and your search return incomplete results
       # You might want to change this to a higher value (and ensure your host has enough memory)
       MAX_MEMORY_USAGE_RATIO: 0.3
+      CLICKHOUSE_LOGGER_LEVEL: information
```

```suggestion
      CLICKHOUSE_LOGGER_LEVEL: 'information'
```

Let's see if adding quotes around this fixes the issue. If this doesn't fix it, let's just hard-code `information` into the XML file for now. I've Googled this error to no avail.